src/trg-preferences-dialog.c

```diff
@@ -888,6 +891,22 @@ static GtkWidget *trg_prefs_serverPage(TrgPreferencesDialog * dlg)
                       TRG_PREFS_PROFILE, NULL);
     hig_workarea_add_row(t, &row, _("Retries:"), w, NULL);
 
+    hig_workarea_add_section_title (t, &row, _ ("Headers"));
```

Comment:
Only style thing, no space after `(` for functions.

Reply:
Do you have a .clang-format that I can use?

Reply:
No.

Reply:
Can you point to a standard I should use? I am a bit confused because there is no clear standard and when I patch some lines, some have tabs and others space, so I can at least make it standard within my patch.